SDWebImage/SDWebImageManager.h

```diff
@@ -198,46 +210,13 @@ SDWebImageManager.sharedManager.cacheKeyFilter = ^(NSURL * _Nullable url) {
                                              progress:(nullable SDWebImageDownloaderProgressBlock)progressBlock
                                             completed:(nonnull SDInternalCompletionBlock)completedBlock;
 
-/**
- * Saves image to cache for given URL
- *
- * @param image The image to cache
- * @param url   The URL to the image
- *
- */
-
-- (void)saveImageToCache:(nullable UIImage *)image forURL:(nullable NSURL *)url;
-
 /**
  * Cancel all current operations
  */
 - (void)cancelAll;
 
 /**
- *  Async check if image has already been cached
```

Comment:
Why to remove these 3 methods because for `manager` it allow to get back the specify cache object. And these 3 methods just call `cacheKeyForURL:` to get the `cacheKey`, then call the cache object's method. It's really easy to do the same thing by your own. However, put these into manager's public API cause things difficult and tied manager with cache's implementation. So it's time to clean up.